src/main/java/com/wanderersoftherift/wotr/datagen/ModModelProvider.java

```diff
@@ -93,4 +80,14 @@ public void generateRunegemItem(Item item, ItemModelGenerators itemModels) {
         itemModels.itemModelOutput.accept(item, ItemModelUtils.select(new SelectRuneGemShape(), ItemModelUtils.plainModel(modelLocation), list));
     }
 
+    public static PropertyDispatch createOppositeFacingDispatch() {
```

Comment:
Did you check if something like this does not already exist for other facingblocks, like wood?

Reply:
There is an oppositeDispatch, but it assumes the initial model faces NORTH. In this case the initial model faces UP.